hm_1.py

```diff
@@ -0,0 +1,5 @@
+num = input('Num: ')
+suma = 0
+for i in num:
+    suma += int(i)
```

Comment:
It might be calculated simpler as `Zen of Python` recommends to us:
https://www.python.org/dev/peps/pep-0020/

Reply:
I will take note